Introduction to Federated Learning with Substra.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<p>The numb update, round, index generator etc... can be quite hard to understand. There is this ressource on the doc, with a schema, that can be helpful to understand this part :) https://docs.substra.org/en/stable/substrafl_doc/substrafl_overview.html#index-generator</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-workshops/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>There has been some discussion on whether we want to spend more time on this concept or not, and in particular including the diagram that might trigger more questions at first. </p><p>But maybe adding the link to the resource is a good compromise, indeed :+1:</p>